File: 1-chapter/Part_2/fibonacciByInteration.js
function fibonacci(number) {
    let a = 1, b = 1;
    let nextNum;
    console.log('a =', a, 'b=', b)
    for (let i = 1; i < number; i++) {
        nextNum = a + b;
        a = b;
        b = nextNum;
        console.log('a =', a, 'b=', b)
    }
}

fibonacci(10)

File: 1-chapter/Part_2/fibonacciByInteration.py
def fibonacci(number):
    a, b = 1, 1
    print('a = %s, b = %s' % (a, b))
    for i in range(1, number):
        a, b = b, a + b
        print('a = %s, b = %s' % (a, b))
    return a


print(fibonacci(10))
